official/requirements.txt

```diff
@@ -0,0 +1,2 @@
+psutil>=5.4.3
+py-cpuinfo>=3.3.0
```

Comment:
Should tf-nightly be listed here too? I don't know if there's a way to do a conditional on gpu.